src/pygaps/modelling/__init__.py

```diff
@@ -45,6 +47,8 @@
     "BET",
     "TemkinApprox",
     "Toth",
+    "DSToth",
```

Comment:
As the DSToth and ChemiPhysisorption methods are fairly advanced, I would not automatically run them.

Reply:
[dd988c9](https://github.com/pauliacomi/pyGAPS/pull/54/commits/dd988c95b4b4dab2547c61c08b8836b8d193cb7b)